ArmaForces.Boderator/ArmaForces.Boderator.BotService/Controllers/DLCController.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using System.IO;
+using ArmaForces.Boderator.BotService.DataClasses;
+
+namespace ArmaForces.Boderator.BotService.Controllers
+{
+    [Route("users/")]
+    [ApiController]
+    [Produces("application/json")]
```

Comment:
This is redundant, you can remove this.
```suggestion
```

Reply:
Redundant with what? Something in rest of program?

Reply:
You don't need to specify what `Content-Type` you produce if you just use `IActionResult` and return the data object. It will be serialized to any `Content-Type` desired by caller.